spec/suites/layer/vector/PolygonSpec.js

```diff
@@ -175,6 +184,25 @@ describe('Polygon', function () {
 			}).to.throwException('Must add layer to map before using getCenter()');
 		});
 
+		it('should compute same center for low and high zoom', function () {
+			var latlngs = [
+				[[0, 0], [0.010, 0], [0.010, 0.010], [0, 0.010]]
+			];
+			var layer = new L.Polygon(latlngs).addTo(map);
+			map.setZoom(0);
+			var center = layer.getCenter();
+			map.setZoom(18);
+			expect(layer.getCenter()).to.be.nearLatLng(center);
+		});
+
+		it("should compute center for multi-polygon including hole", function () {
+			var latlngs = [
+				[[[10, 20], [30, 40], [50, 60]]],
+				[[[0, 10], [10, 10], [10, 0]], [[2, 3], [2, 4], [3, 4]]]
+			];
+			var layer = new L.Polygon(latlngs).addTo(map);
+			expect(layer.getCenter()).to.be.nearLatLng(L.latLng([31.43, 39.99]), 1e-1);
```

Comment:
Why are you using different precision values?
What if you stick to default?

Reply:
The latlng values are long numbers but I replaced it now with the exact number, it is cleaner